src/providers/MeetingProvider/docs/MeetingProvider.stories.mdx

```diff
@@ -32,6 +32,16 @@ const App = () => (
 
 2. Render two `MeetingProvider`s near the root of your application with the same `MeetingManager` instance.
 
+**IMPORTANT NOTE**
+This approach has limitations. This should be used only in very specific cases where you want to share the meeting manager instance
+variable values or you will not use video, roster and audio at both places. Still, the hooks and providers may not work as expected as `MeetingProvider` wraps all the other providers.
+The video will not work due to limitations of JS SDK to bind a video stream to a single video tile only. Please check [this](https://github.com/aws/amazon-chime-sdk-component-library-react/issues/492#issuecomment-846317339) issue for more details.
+
+If you are using the same `attendeeInfo`, then please make sure your app calls `meetingManager.join()` only once for that `attendeeInfo`. 
```

Comment:
This is a bit long. How about:
> Please make sure your app calls `meetingManager.join()` only once with the same `attendeeInfo`. Otherwise,  the previous attendee who joined the meeting will leave the meeting with [AudioJoinedFromAnotherDevice](https://aws.github.io/amazon-chime-sdk-js/enums/meetingsessionstatuscode.html#audiojoinedfromanotherdevice).

Reply:
So, no explicit saying for different attendeeId?

Reply:
Updated it to what you said which is more crispier :)